### PR TITLE
fix(chat): move runtime selector to separate field (Issues #2478, #2515)

### PR DESCRIPTION
**Pull request by Alexander-Kezik**
**Description:**

move runtime selector to separate field

Issues:

- https://github.com/epam/ai-dial-chat/issues/2478
- #2515

**Checklist:**

- [x] the pull request name complies with [Conventional Commits](https://www.conventionalcommits.org/en/v1.0.0/)
- [x] the pull request name starts with `fix(<scope>):`, `feat(<scope>):`, `feature(<scope>):`, `chore(<scope>):`, `hotfix(<scope>):` or `e2e(<scope>):`. If contains breaking changes then the pull request name must start with `fix(<scope>)!:`, `feat(<scope>)!:`, `feature(<scope>)!:`, `chore(<scope>)!:`, `hotfix(<scope>)!:` or `e2e(<scope>)!:` where `<scope>` is name of affected project: `chat`, `chat-e2e`, `overlay`, `shared`, `sandbox-overlay`, etc.
- [x] the pull request name ends with `(Issue #<TICKET_ID>)` (comma-separated list of issues)
- [x] I confirm that do not share any confidential information like API keys or any other secrets and private URLs

### CODE REVIEW

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2519.nightly-test.deltixhub.io
>E2E tests status: success

File: apps/chat/src/components/Common/ApplicationWizard/CodeAppView/RuntimeVersionSelector.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+              'absolute right-3 top-1/2 shrink-0 -translate-y-1/2 transition-all',
+              isVersionSelectorOpen && 'rotate-180',
+            )}
+            size={18}
+          />
+        </div>
+      }
+    >
+      {pythonVersions.map((version) => {
+        if (version === value) return null;
```
Comment (IlyaBondar) [inline apps/chat/src/components/Common/ApplicationWizard/CodeAppView/RuntimeVersionSelector.tsx]: we need to show selected option I suppose

**Review by IlyaBondar — DISMISSED** [review]

**Comment by Alexander-Kezik** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2519.nightly-test.deltixhub.io
>E2E tests status: failed

**Review by IlyaBondar — APPROVED** [review]

**Comment by nartovm** [comment]
/deploy-review
>Environment URL: https://chat-ai-dial-chat-pr-2519.nightly-test.deltixhub.io
>E2E tests status: success

**Pull request merged**